satellite/marketing/offers_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information
+
+package marketing_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testplanet"
+	"storj.io/storj/satellite/marketing"
+)
+
+func TestOffer_Database(t *testing.T) {
+	testplanet.Run(t, testplanet.Config{
+		SatelliteCount: 1,
+	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+		// Happy path
+		validOffers := []marketing.NewOffer{
+			{
+				Name:                      "test",
+				Description:               "test offer 1",
+				AwardCreditInCents:        100,
+				InviteeCreditInCents:      50,
+				AwardCreditDurationDays:   60,
+				InviteeCreditDurationDays: 30,
+				RedeemableCap:             50,
+				ExpiresAt:                 time.Now().UTC().Add(time.Hour * 1),
+				Status:                    marketing.Active,
+				Type:                      marketing.Referral,
+			},
+			{
+				Name:                      "test",
+				Description:               "test offer 2",
+				AwardCreditInCents:        100,
+				InviteeCreditInCents:      50,
+				AwardCreditDurationDays:   60,
+				InviteeCreditDurationDays: 30,
+				RedeemableCap:             50,
+				ExpiresAt:                 time.Now().UTC().Add(time.Hour * 1),
+				Status:                    marketing.Default,
+				Type:                      marketing.FreeCredit,
+			},
+		}
+
+		for i := range validOffers {
```

Comment:
nit: this can be also `for _, validOffer := range validOffers {`

Reply:
I was thinking that since `validOffers` is a reference type, it's better to not create copies of the pointer and just use the original pointer value in the slice

Reply:
sounds good :+1: